aladdin/lib/arg_tools.py

```diff
@@ -4,12 +4,46 @@
 import subprocess
 import sys
 import pkg_resources
+from contextlib import suppress
+from inspect import signature
+
+from kubernetes import config as kube_config
 
 from aladdin.config import PROJECT_ROOT
 
 
+def get_current_namespace():
+    namespace = os.getenv("NAMESPACE")
+
+    if not namespace:
+        with suppress(KeyError, kube_config.config_exception.ConfigException):
+            _, active_context = kube_config.list_kube_config_contexts()
+            namespace = active_context["context"]["namespace"]
+
+    if not namespace:
+        namespace = "default"
+    return namespace
+
+
+def expand_namespace(func=None):
+    """
+    Decorator to expand a Namespace obj into keyworded arguments
+    """
```

Comment:
Maybe let's move this out from in between two other functions that deal with the kubernetes namespace value.

Reply:
moved :+1: